Hudl.Mjolnir.Attributes/CommandAttribute.cs

```diff
@@ -15,10 +15,23 @@ public sealed class CommandAttribute : Attribute
         private readonly string _breakerKey;
         private readonly string _poolKey;
         private readonly int _timeout;
+        private readonly bool _ignoreTimeout;
 
         // See Mjolnir's Command constructors.
         public CommandAttribute(string group, string isolationKey, int timeout = DefaultTimeout) : this(group, isolationKey, isolationKey, timeout) { }
 
+        public CommandAttribute(string group, string isolationKey, bool ignoreTimeout)
```

Comment:
I've added this extra constructor to allow for setting specific Command calls to ignore timeouts. Is this wise? Do we want to expose this feature?